core/primitives/src/trie_key.rs

```diff
@@ -56,6 +56,7 @@ pub mod col {
     /// backpressure on the receiving shard.
     /// (`primitives::receipt::Receipt`).
     pub const BUFFERED_RECEIPT: u8 = 14;
+    pub const BANDWIDTH_SCHEDULER_STATE: u8 = 15;
```

Comment:
I have some bad news for you. Since this is the last single byte value available we will need to use it to introduce two-byte type prefix. Please see the NOTE three lines below here. 

Probably best done separately, I would not object to leaving it as is for now and following up. Just please make sure it doesn't leak to mainnet or we'll be in for a lot of pain undoing it.

Reply:
nit: Add a comment please.

Reply:
A byte can contain up to 255, so the u8 should last a while, no need for two bytes.
But the next column will have two nibbles, the Trie should be able to handle it, right?

Reply:
Ah sorry, I mixed that up with nibbles. You're right, we should be just fine. That's awesome :)